### DISCUSSION TITLE
XMPDump

### DISCUSSION THREAD

**Discussion by tester0077** [jim-easterbrook/python-exiv2#43] [Q&A] [answered]
For Windows and XMP, Exiv2 has/had a utility XMPDump.
Is there anything equivalent as part of the Python library for Exiv2?
Or a way to make the the code give similar results?
TIA

**Comment by jim-easterbrook** [comment]
`xmpData().xmpPacket()` returns the raw XMP string. Is that what you need?

**Comment by tester0077** [comment]
Yes, thank you.
My apologies for the delayed response.
Are there any options to serialize the string for better legibility?

**Reply by jim-easterbrook** [reply, marked as answer]
The Python standard library has XMP parsers that should be able to parse it and then regenerate it in a prettified format.
